src/editor/actions/local-autosave-actions.js

```diff
@@ -0,0 +1,44 @@
+import { getLocalAutosave } from "../tools/local-autosave";
```

Comment:
as above: can you annotate this file to indicate which parts are just things moved over from actions.js and what is new?